### PR TITLE
[Grafana] Add relation grafana dashboard.

### PR DESCRIPTION
**Pull request by yswdqz**

### CODE REVIEW

**Comment by wankai123** [comment]
Have you tested the template on showcase？ You can test it on the docker model.
This dashboard is only for layer `GENERAL`, so I suggest renaming it.

File: deploy/platform/config/grafana/dashboards/relation.json [hunk 1]
```diff
@@ -0,0 +1,2256 @@
...
+          "value": "fdoqp17pgc9ogc"
+        },
+        "hide": 2,
+        "includeAll": false,
+        "multi": false,
+        "name": "DS_SW_Prometheus",
+        "options": [],
+        "query": "prometheus",
+        "refresh": 1,
+        "regex": "prometheus",
```
Comment (wankai123) [inline deploy/platform/config/grafana/dashboards/relation.json:2145]:
```suggestion
        "regex": "SkyWalkingPromQL",
```
require this pattern to find the existing DS https://github.com/apache/skywalking-showcase/blob/d949cef04b5e3c15adcaf435cfdd4cb20e9b5eaa/deploy/platform/docker/config/grafana/datasources.yaml#L24

File: deploy/platform/config/grafana/dashboards/relation.json [hunk 2]
```diff
@@ -0,0 +1,2256 @@
...
+        "skipUrlSync": false,
+        "type": "datasource"
+      },
+      {
+        "allValue": "*",
+        "current": {
+          "selected": true,
+          "text": "e2e-service-provider",
+          "value": "e2e-service-provider"
```
Comment (wankai123) [inline deploy/platform/config/grafana/dashboards/relation.json:2154]: I think you should run it on the showcase and re-export the template to correct the default value.

**Comment by yswdqz** [comment]
image

**Review by wankai123 — APPROVED** [review]

**Pull request merged**